plotly_express/_core.py

```diff
@@ -133,6 +133,8 @@ def make_trace_kwargs(args, trace_spec, g, mapping_labels, sizeref, color_range)
                     or len(args["data_frame"][name].unique()) <= 20
                 )
             ]
+        elif k == 'opacity':
```

Comment:
oh you can remove this if you don't add opacity to `attrables`

Reply:
If I remove this, it attempts to use the value of opacity as a column name to find in the dataframe and set according to it's values... so I'm not sure how else to go about it?
```
KeyError: 0.1
...
    228             else:
    229                 if v:
--> 230                     result[k] = g[v]
    231                 mapping_labels.append(("%s=%%{%s}" % (v_label, k), None))
    232     if trace_spec.constructor not in [
```

Reply:
So it was actually pretty simple haha...